versioned_docs/version-23.10/monitoring/basic-objects/hosts.md

```diff
@@ -52,82 +67,76 @@ This button is used to delete the host template ![image](../../assets/configurat
-* The **Recovery notification delay** is the time that must pass before a recovery notification is sent (when the host goes back to an UP state).
+* **Notification Options**: define the statuses for which a notification will be sent. If no boxes are checked, you will receive notifications for all the statuses listed.
+* **Notification Interval**: expressed in minutes. It indicates the time between sending notifications when the status is Not-OK. If the value is defined as 0, the scheduler sends a single notification per status change.
+  If the value is defined as 0, the scheduler sends a single notification per status change.
+* **Notification Period**: indicates the time period during which the notifications will be enabled.
+* **First notification delay**: expressed in minutes. It refers to the time delay to be respected before sending
+  the first notification when a Not-OK status is validated. Setting the value to 0 will cause notifications to be sent immediately.
+* **Recovery notification delay**: time that must pass before a recovery notification is sent (when the host goes back to an UP state). Setting the value to 0 will cause notifications to be sent immediately.
 
 ## Relations tab
+Also called [dependencies](../../alerts-notifications/notif-dependencies.md)
 
-* The **Host Groups** list defines the host groups to which the host belongs.
-* The **Host Categories** list defines the categories to which the host belongs.
-* The **Parent Hosts** list enables us to define the physical family relationships between objects.
-* The **Child Hosts** list enables us to define the physical family relationships between objects.
+* **Host Groups**: host groups to which the host belongs. Groups allow you to make changes to many hosts at the same time.
+* **Host Categories**: categories to which the host belongs.
+* **Parent Hosts**: define an object as a parent for this host. The parent of a host is the object between it and its monitoring host that is also the closest to the monitored object. A host is considered to have no parent host if it is on the same network segment as the host doing the monitoring without any intermediates. Leave this field blank if that is the case.
+* **Child Hosts**: designate the current host as the parent for other hosts.
 
 ## Data processing tab
 
-* If **Obsess Over Host** is enabled, the host check feedback command will be enabled.
-* The **Check Freshness** field allows us to enable or disable the result freshness check.
-* The **Freshness Threshold** is expressed in seconds. If, during this period, no host status change request (passive
-  command) is received, the active check command is executed.
-* The **Flap Detection Enabled** field allows us to enable or disable the detection flapping in the statuses (status
+* **Check Freshness**: active check performed by the engine when a determined threshold of time has elapsed since the last [passive check](../../passive-monitoring/enable-snmp-traps.md) of the object.
```

Comment:
```suggestion
* **Check Freshness**: active check performed by the engine when a determined threshold of time has elapsed since the last [passive check](../../monitoring/passive-monitoring/enable-snmp-traps.md) of the object.
```